types/eslint-utils/package.json

```diff
@@ -0,0 +1,3 @@
+{
+    "private": true
+}
```

Comment:
Please remove the file completely after already addressing previous comment.
thx!